### PR TITLE
docs: add examples for random_uniform, random_normal and random_binomial in tf.keras.backend

### LINKED ISSUE #31277: [TF 2.0 API Docs] tf.keras.backend.random_uniform
## URL(s) with the issue:
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/backend/random_uniform

## Description of issue (what needs changing):
The function has no usage example and there are no errors raised.

### Raises listed and defined
No raises listed or defined.

### Usage example
No usage example

### Submit a pull request?
No.

### PR DESCRIPTION
**Pull request by Ir1d**
closes #31277

### CODE REVIEW

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: tensorflow/python/keras/backend.py [hunk 1]
```diff
@@ -5653,6 +5653,13 @@ def random_normal(shape, mean=0.0, stddev=1.0, dtype=None, seed=None):
 
   Returns:
       A tensor with normal distribution of values.
+
+  Example:
+
+  >>> kvar = tf.keras.backend.random_normal((2,3), 0, 1)
+  >>> kvar
+  <tf.Tensor: shape=(2, 3), dtype=float32, numpy=...,
```
[the remark this answers is not in the document]
Comment (Ir1d) [reply tensorflow/python/keras/backend.py]: Hi, when testing locally, I noticed that even if I specify a fixed seed, the  result still varies from time to time.
Also, the examples in `random_normal_variable` and `random_uniform_variable` checked only the shape and type of the result as well.

**Comment by Ir1d** [comment]
Hi @ql I updated the PR as you suggested in the comments. I've also updated the examples for `random_uniform_variable` and `random_normal_variable`, because previously I was trying to following the convention.

**Comment by Ir1d** [comment]
Hi @qlzh727 I updated the PR as you suggested in the comments. I've also updated the examples for `random_uniform_variable` and `random_normal_variable`, because previously I was trying to following the convention.

**Review by qlzh727 — DISMISSED** [review]

**Comment by qlzh727** [comment]
Thanks for the update.

**Comment by gbaned** [comment]
@Ir1d Can you please address Ubuntu Sanity errors? Thanks!

**Comment by Ir1d** [comment]
@gbaned I've solved the pylint issue, it should be ready now.

**Comment by jaketae** [comment]
CI build returns the following error message. Posting it here as a friendly reference for others!
```
Traceback (most recent call last):
  File "/usr/lib/python3.6/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.6/unittest/case.py", line 605, in run
    testMethod()
  File "/usr/lib/python3.6/doctest.py", line 2199, in runTest
    raise self.failureException(self.format_failure(new.getvalue()))
AssertionError: Failed doctest test for tensorflow.python.keras.backend.random_binomial
  File "/b/f/w/bazel-out/k8-opt/bin/tensorflow/tools/docs/tf_doctest.runfiles/org_tensorflow/tensorflow/python/keras/backend.py", line 316, in random_binomial

----------------------------------------------------------------------
File "/b/f/w/bazel-out/k8-opt/bin/tensorflow/tools/docs/tf_doctest.runfiles/org_tensorflow/tensorflow/python/keras/backend.py", line 340, in tensorflow.python.keras.backend.random_binomial
Failed example:
    random_binomial_tensor = tf.keras.backend.random_binomial(shape=(2,3),
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python3.6/doctest.py", line 1330, in __run
        compileflags, 1), test.globs)
      File "<doctest tensorflow.python.keras.backend.random_binomial[0]>", line 1
        random_binomial_tensor = tf.keras.backend.random_binomial(shape=(2,3),
                                                                             ^
    SyntaxError: unexpected EOF while parsing
----------------------------------------------------------------------
File "/b/f/w/bazel-out/k8-opt/bin/tensorflow/tools/docs/tf_doctest.runfiles/org_tensorflow/tensorflow/python/keras/backend.py", line 342, in tensorflow.python.keras.backend.random_binomial
Failed example:
    random_binomial_tensor
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python3.6/doctest.py", line 1330, in __run
        compileflags, 1), test.globs)
      File "<doctest tensorflow.python.keras.backend.random_binomial[1]>", line 1, in <module>
        random_binomial_tensor
    NameError: name 'random_binomial_tensor' is not defined
```

File: tensorflow/python/keras/backend.py [hunk 2]
```diff
@@ -1479,7 +1479,8 @@ def random_uniform_variable(shape, low, high, dtype=None, name=None, seed=None):
 
   Example:
 
-  >>> kvar = tf.keras.backend.random_uniform_variable((2,3), 0, 1)
+  >>> kvar = tf.keras.backend.random_uniform_variable(shape=(2,3),
+  low=0.0, high=1.0)
```
Comment (qlzh727) [inline tensorflow/python/keras/backend.py]: I think this is breaking tf_docstring test. You shouldn't wrap the line here.

File: tensorflow/python/keras/backend.py [hunk 3]
```diff
@@ -1513,7 +1514,8 @@ def random_normal_variable(shape, mean, scale, dtype=None, name=None,
 
   Example:
 
-  >>> kvar = tf.keras.backend.random_normal_variable((2,3), 0, 1)
+  >>> kvar = tf.keras.backend.random_normal_variable(shape=(2,3),
+  mean=0.0, scale=1.0)
```
Comment (qlzh727) [inline tensorflow/python/keras/backend.py]: same.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: tensorflow/python/keras/backend.py (hunk 2 above)
Comment (Ir1d) [reply tensorflow/python/keras/backend.py]: Hi, I've added prepending `... ` so that doctest could recognize it.

**Review by qlzh727 — APPROVED** [review]

**Pull request merged**